hack/test-update-storage-objects.sh

```diff
@@ -107,16 +107,17 @@ echo "${ETCD_VERSION}/${STORAGE_BACKEND_ETCD2}" > "${ETCD_DIR}/version.txt"
 
 ### BEGIN TEST DEFINITION CUSTOMIZATION ###
 
+# After removing HPA in v1beta1 there is basically no resource which has
+# two versions. In the future we may want to extend it.
```

Comment:
Jobs should be removed from v2alpha1 per https://github.com/kubernetes/kubernetes/issues/37166